pylsp/python_lsp.py

```diff
@@ -399,6 +474,83 @@ def folding(self, doc_uri):
     def m_completion_item__resolve(self, **completionItem):
         return self.completion_item_resolve(completionItem)
 
+    def m_notebook_document__did_open(self, notebookDocument=None, cellTextDocuments=None, **_kwargs):
+        workspace = self._match_uri_to_workspace(notebookDocument['uri'])
+        workspace.put_notebook_document(notebookDocument['uri'], notebookDocument['notebookType'],
+                                        cells=notebookDocument['cells'], version=notebookDocument.get('version'),
+                                        metadata=notebookDocument.get('metadata'))
```

Comment:
```suggestion
        workspace.put_notebook_document(
            notebookDocument['uri'],
            notebookDocument['notebookType'],
            cells=notebookDocument['cells'],
            version=notebookDocument.get('version'),
            metadata=notebookDocument.get('metadata')
        )
```

Stylistic fix to make this look better.

Reply:
That looks better for me too. However, I didn't want to break the styling pattern to other functions. Looks more like black btw :)